lib/Driver/ToolChains.cpp

```diff
@@ -1459,9 +1459,11 @@ void ToolChain::getClangLibraryPath(const ArgList &Args,
   getResourceDirPath(LibPath, Args, /*Shared=*/true);
   // Remove platform name.
   llvm::sys::path::remove_filename(LibPath);
-  llvm::sys::path::append(LibPath, "clang", "lib",
-                          T.isOSDarwin() ? "darwin"
-                                         : getPlatformNameForTriple(T));
+  auto platformName =
+      T.isOSDarwin() ? "darwin" : getPlatformNameForTriple(Triple);
+  if (platformName == "android")
```

Comment:
Would you mind using `T.isOSAndroid()` instead for this?

Reply:
Sure, `T.isAndroid()` is equivalent, fine by me, will make that change.

Reply:
@compnerd, made this change, let me know if it's okay in latest commit.